core/src/test/java/io/questdb/griffin/SqlCodeGeneratorTest.java

```diff
@@ -203,6 +203,238 @@ public void testAnalyticFunctionWithFilter() throws Exception {
         );
     }
 
```

Comment:
Would be great to add a test with multiple columns in the OVER(... ORDER BY) clause.